packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/tasks/ConfigureModelResponseHandler.java

```diff
@@ -69,20 +70,24 @@ public void onSuccess(TaskResponse resp) {
 			final Model model = modelService.getAsset(props.getModelId())
 					.orElseThrow();
 			final Response configurations = objectMapper.readValue(((TaskResponse) resp).getOutput(), Response.class);
-
-			// For each configuration, create a new model configuration with parameters set
-			for (JsonNode condition : configurations.response.get("conditions")) {
-				// Map the parameters values to the model
-				final Model modelCopy = new Model(model);
-				final List<ModelParameter> modelParameters = modelCopy.getSemantics().getOde().getParameters();
-				modelParameters.forEach((parameter) -> {
-					JsonNode conditionParameters = condition.get("parameters");
-					conditionParameters.forEach((conditionParameter) -> {
-						if (parameter.getId().equals(conditionParameter.get("id").asText())) {
-							parameter.setValue(conditionParameter.get("value").doubleValue());
-						}
-					});
-				});
+            // For each configuration, create a new model configuration with parameters set
+            for (JsonNode condition : configurations.response.get("conditions")) {
+                // Map the parameters values to the model
+                final Model modelCopy = new Model(model);
+                 List<ModelParameter> modelParameters;
+					if(modelCopy.getHeader().getSchemaName().toLowerCase().equals("regnet")) {
```

Comment:
Is this a weird github thing or is there wonky indenting here?

Reply:
it was a little off, but githubs making it look a lot worse 😆 
<img width="995" alt="Screenshot 2024-03-01 at 9 43 00 AM" src="https://github.com/DARPA-ASKEM/terarium/assets/33158416/14b5a35c-acac-4d3a-8a40-9d25b3566660">

Just fixed that up